cluster/images/crossplane/Makefile

```diff
@@ -12,6 +12,10 @@ include ../../../build/makelib/image.mk
 # ====================================================================================
 # Targets
 
+ifeq ($(PLATFORM),$(filter $(PLATFORM),darwin_amd64 windows_amd64)) 
+$(info Skipping image build for $(PLATFORM))
+img.build:
+else
```

Comment:
I don't _love_ this, but I think I actually prefer it to doing a larger refactor in the `build` submodule or trying to work around it in our main `Makefile`. You can see that the correct [images](https://releases.crossplane.io/build/PR-1746/v0.13.0-rc.153.gd2399b5/images/) and [binaries](https://releases.crossplane.io/build/PR-1746/v0.13.0-rc.153.gd2399b5/bin/) have been built for this PR.

Reply:
I don't really know the details of `Makefile` syntax. Is it possible to have the if/else under `img.build` and print `Skipping image build since the target is ...` in case it's skipped?

Reply:
It is possible to exit earlier, but from what I could tell while trying to determine the best way to do this, it is considered better practice to override the target